src/components/WalletConnect.vue

```diff
@@ -50,6 +51,25 @@ export default defineComponent({
 			WalletType,
 		};
 	},
+	mounted() {
+		searchForAssets("gov-token")
```

Comment:
From where do we get the name `"gov-token"`? Is the ASA Name (always constant)? If so, maybe we should keep it somewhere in `constants.ts` ?

Reply:
ok moved in constant file